test_upload_from_file_w_bound_client_multipart fails on Windows

Introduced in #1318

My [original comment][1] points to a [broken build][2].

```
```
AssertionError: Lists differ: ['full_name,age,joined', '', 'Phred Phlynt[185 chars]27,'] != [u'full_name,age,joined', u'', u'Phred Phl[199 chars] u'']
 
Second list contains 1 additional elements.
First extra element 9:
 
 
- ['full_name,age,joined',
+ [u'full_name,age,joined',
?  +
 
-  '',
+  u'',
?  +
 
-  'Phred Phlyntstone,32,2015-07-24 19:53:19.006000+00:00',
+  u'Phred Phlyntstone,32,2015-07-24 19:53:19.006000+00:00',
?  +
 
-  '',
+  u'',
?  +
 
-  'Bharney Rhubble,33,2015-07-24 19:53:20.006000+00:00',
+  u'Bharney Rhubble,33,2015-07-24 19:53:20.006000+00:00',
?  +
 
-  '',
+  u'',
?  +
 
-  'Wylma Phlyntstone,29,2015-07-24 19:53:21.006000+00:00',
+  u'Wylma Phlyntstone,29,2015-07-24 19:53:21.006000+00:00',
?  +
 
-  '',
+  u'',
?  +
 
-  'Bhettye Rhubble,27,']
?                       ^
 
+  u'Bhettye Rhubble,27,',
?  +                     ^
 
+  u'']
 
```

[1]: https://github.com/GoogleCloudPlatform/gcloud-python/pull/1318#issuecomment-176942213
[2]: https://ci.appveyor.com/project/dhermes/gcloud-python/build/1.0.54.master

--- a/gcloud/bigquery/test_table.py
+++ b/gcloud/bigquery/test_table.py
@@ -1397,9 +1397,9 @@ class TestTable(unittest2.TestCase, _SchemaBase):
                          {'Content-Type': 'application/octet-stream',
                           'Content-Transfer-Encoding': 'binary',
                           'MIME-Version': '1.0'})
-        body = BODY.decode('ascii')
+        body = BODY.decode('ascii').rstrip()
         body_lines = [line.strip() for line in body.splitlines()]
-        payload_lines = app_msg._payload.splitlines()
+        payload_lines = app_msg._payload.rstrip().splitlines()
         self.assertEqual(payload_lines, body_lines)
 
     # pylint: disable=R0915

